### PR TITLE
[docs] mention different levels of update commands

### PR DESCRIPTION
**Pull request by fearful-symmetry**
This came out of a discussion on slack. The root `update` command is a tad non-obvious, and errors related to the `notice.txt` changes that happen when a dependency is added or removed is also _very_ non-obvious.

### CODE REVIEW

File: docs/devguide/contributing.asciidoc [hunk 1]
```diff
@@ -96,6 +96,8 @@ and documentation. The primary command used for this is:
 --------------------------------------------------------------------------------
 make update
 --------------------------------------------------------------------------------
+Each beat has its own `update` target, as well as a master `update` in the repository root.
```
Comment (dedemorton) [inline docs/devguide/contributing.asciidoc]:
```suggestion
Each Beat has its own `update` target, as well as a master `update` in the repository root. 
```

File: docs/devguide/contributing.asciidoc [hunk 2]
```diff
@@ -96,6 +96,8 @@ and documentation. The primary command used for this is:
 --------------------------------------------------------------------------------
 make update
 --------------------------------------------------------------------------------
+Each beat has its own `update` target, as well as a master `update` in the repository root. 
+If a PR adds or removes a dependency, `update` should be run in both the individual beat directory, and the root `beats` directory.
```
Comment (dedemorton) [inline docs/devguide/contributing.asciidoc]:
```suggestion
If a PR adds or removes a dependency, run `update` in both the individual Beat directory and the root `beats` directory.
```

Comment (dedemorton) [reply docs/devguide/contributing.asciidoc]: This surprises me a bit. I thought you could run `make update` in the root to update everything. What sorts of dependencies require you to run in the individual directories vs the root?

Comment (fearful-symmetry) [reply docs/devguide/contributing.asciidoc]: Ah, you're actually right. The root `make update` runs its own scripts as well as the individual beats ones.

**Comment by fearful-symmetry** [comment]
jenkins, test this

**Comment by fearful-symmetry** [comment]
jenkins, test this

**Comment by fearful-symmetry** [comment]
jenkins, test this

**Pull request merged**